content/webapp/components/SearchResults/SearchResults.tsx

```diff
@@ -47,218 +48,100 @@ const SearchResults: FunctionComponent<Props> = ({
-                url={linkResolver(item)}
-                title={item.title || ''}
-                primaryLabels={item.labels}
-                secondaryLabels={[]}
-                description={item.promo && item.promo.caption}
-                urlOverride={item.promo && item.promo.link}
-                Image={
-                  getCrop(item.image, 'square') && (
-                    <PrismicImage
-                      image={{
-                        // We intentionally omit the alt text on promos, so screen reader
-                        // users don't have to listen to the alt text before hearing the
-                        // title of the item in the list.
-                        //
-                        // See https://github.com/wellcomecollection/wellcomecollection.org/issues/6007
-                        ...getCrop(item.image, 'square')!,
-                        alt: '',
-                      }}
-                      sizes={{
-                        xlarge: 1 / 6,
-                        large: 1 / 6,
-                        medium: 1 / 5,
-                        small: 1 / 4,
-                      }}
-                      quality="low"
-                    />
-                  )
-                }
-                xOfY={{ x: index + 1, y: items.length }}
-              />
-            )}
-            {item.type === 'events' && (
-              <EventCard
-                event={item}
-                xOfY={{ x: index + 1, y: items.length }}
-              />
-            )}
-            {item.type === 'exhibitions' && (
-              <CompactCard
-                url={linkResolver(item)}
-                title={item.title}
-                primaryLabels={item.labels}
-                secondaryLabels={[]}
-                description={item.promo?.caption}
-                Image={
-                  getCrop(item.image, 'square') && (
-                    <PrismicImage
-                      image={{
-                        // We intentionally omit the alt text on promos, so screen reader
-                        // users don't have to listen to the alt text before hearing the
-                        // title of the item in the list.
-                        //
-                        // See https://github.com/wellcomecollection/wellcomecollection.org/issues/6007
-                        ...getCrop(item.image, 'square')!,
-                        alt: '',
-                      }}
-                      sizes={{
-                        xlarge: 1 / 6,
-                        large: 1 / 6,
-                        medium: 1 / 5,
-                        small: 1 / 4,
-                      }}
-                      quality="low"
-                    />
-                  )
-                }
-                xOfY={{ x: index + 1, y: items.length }}
-              />
-            )}
-          </Result>
-        ))}
+        {items.map((item, index) => {
+          if (item.type === 'weblinks') return null;
```

Comment:
This type did not contain `title`. As we don't render it anyway, I return here to simplify the code below and the typescript flags it would've created.